### PR TITLE
Addition to changelog entry from 1.4.4

### PR DESCRIPTION
**Pull request by philippguertler**
## Abstract
Renames duplicate versions instead of deleting them

### General

* [x] Added abstract that describes the change
* [x] Added changelog entry to `/CHANGELOG.adoc`
* [x] Ensured that the change is covered by tests
* [x] Ensured that the change is documented in the docs

### On API Changes

* [x] Checked if the changes are breaking or not
* [x] Added GraphQL API if applicable
* [x] Added Elasticsearch mapping if applicable

### CODE REVIEW

File: core/src/main/java/com/gentics/mesh/changelog/highlevel/change/FixNodeVersionOrder.java [hunk 1]
```diff
@@ -73,6 +74,7 @@ public String getDescription() {
 	public void apply() {
 		AtomicLong nodeCount = new AtomicLong();
 		AtomicLong fixedNodes = new AtomicLong();
+		NodeMigrationActionContextImpl context = new NodeMigrationActionContextImpl();
```
Comment (Jotschi) [inline core/src/main/java/com/gentics/mesh/changelog/highlevel/change/FixNodeVersionOrder.java:77]:
```suggestion
		NodeMigrationActionContext context = new NodeMigrationActionContextImpl();
```

File: core/src/main/java/com/gentics/mesh/changelog/highlevel/change/FixNodeVersionOrder.java [hunk 2]
```diff
@@ -82,19 +84,20 @@ public void apply() {
...
 					});
 				nodeCount.incrementAndGet();
 			});
 		log.info("Checked {} nodes. Found and fixed {} broken nodes", nodeCount.get(), fixedNodes.get());
+		context.getConflicts()
+			.forEach(conflict -> log.info("Encountered conflict for node {" + conflict.getNodeUuid() + "} which was automatically resolved."));
 	}
 
-	private boolean fixNodeVersionOrder(Node node, NodeGraphFieldContainer initialContent) {
+	private boolean fixNodeVersionOrder(NodeMigrationActionContextImpl context, Node node, NodeGraphFieldContainer initialContent) {
```
Comment (Jotschi) [inline core/src/main/java/com/gentics/mesh/changelog/highlevel/change/FixNodeVersionOrder.java:99]:
```suggestion
	private boolean fixNodeVersionOrder(NodeMigrationActionContext context, Node node, NodeGraphFieldContainer initialContent) {
```

File: core/src/main/java/com/gentics/mesh/changelog/highlevel/change/FixNodeVersionOrder.java [hunk 3]
```diff
@@ -112,76 +115,85 @@ private boolean fixNodeVersionOrder(Node node, NodeGraphFieldContainer initialCo
...
-						lower.setNextVersion(currentContainer);
-					}
-					mutated = true;
-				}
-				versions.add(currentContainer);
-				if (currentVersion.getMinor() == 0 && (latestPublished == null || latestPublished.getVersion().getMajor() < currentVersion.getMajor())) {
-					latestPublished = currentContainer;
+			}
+			if (highestVersion != null && currentVersion.compareTo(highestVersion.getVersion()) < 0) {
+				// Wrong order
```
Comment (Jotschi) [inline core/src/main/java/com/gentics/mesh/changelog/highlevel/change/FixNodeVersionOrder.java:134]: Bogus comment?

Comment (philippguertler) [reply core/src/main/java/com/gentics/mesh/changelog/highlevel/change/FixNodeVersionOrder.java:134]: I will clarify the comment

File: core/src/main/java/com/gentics/mesh/changelog/highlevel/change/FixNodeVersionOrder.java (hunk 1 above)
Comment (philippguertler) [reply core/src/main/java/com/gentics/mesh/changelog/highlevel/change/FixNodeVersionOrder.java:77]: The Impl is necessary here to access the conflicts later.

**Pull request merged**